README.md

```diff
@@ -291,7 +291,7 @@ Usage of ./build/bin/darwin/kube2iam:
       --api-token string                    Token to authenticate with the api server
       --app-port string                     Http port (default "8181")
       --auto-discover-base-arn              Queries EC2 Metadata to determine the base ARN
-      --auto-discover-default-role          Queries EC2 Metadata to determine the default Iam Role, cannot be used with --default-role
+      --auto-discover-default-role          Queries EC2 Metadata to determine the default Iam Role and base ARN, cannot be used with --default-role, overwrites any previous setting for --base-role-arn
```

Comment:
This is the output of `kube2iam -h` which uses the description from the flags. I don't think you've modified the description of this flag so that will be overwritten next time someone makes a change. Can you pls update the flag accordingly?